hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/slowoperationdetector/SlowOperationDetector.java

```diff
@@ -184,44 +185,51 @@ private void scanOperationRunner(long nowNanos, long nowMillis, OperationRunner
             }
 
             // create the stack trace once for this operation and cache the invocation for upcoming updates
-            SlowOperationLog log = getOrCreateSlowOperationLog(operationRunner);
-            operationData.invocation = log.getOrCreateInvocation(operationHashCode, durationNanos, nowNanos, nowMillis);
+            SlowOperationLog log = getOrCreateSlowOperationLogOrNull(operationRunner, operation);
+            if (log != null) {
+                int totalInvocations = log.totalInvocations.incrementAndGet();
+                operationData.invocation = log.getOrCreateInvocation(operationHashCode, durationNanos, nowNanos, nowMillis);
 
-            logSlowOperation(operationRunner, log);
+                logSlowOperation(operation, log, totalInvocations);
+            }
         }
 
-        private SlowOperationLog getOrCreateSlowOperationLog(final OperationRunner operationRunner) {
+        private SlowOperationLog getOrCreateSlowOperationLogOrNull(OperationRunner operationRunner, Object operation) {
```

Comment:
The name `getOrCreateSlowOperationLogOrNull` implies a dilemma, if i can get or create, how can i see a null-value-returned?